vllm/v1/core/scheduler.py

```diff
@@ -59,7 +62,7 @@ def __init__(
         # req_id -> Request
         self.requests: Dict[str, Request] = {}
         # Priority queues for requests.
-        self.waiting: Deque[Request] = deque()
+        self.waiting: deque[Request] = deque()
```

Comment:
I guess we should revert this change (make it use `Deque` from `typing` instead of `deque`

Reply:
resolved.